tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -401,3 +401,13 @@ To connect to the docker on SSL, both the docker image and your local machine mu
 In the future, passing in a custom cert via a bound volume that contains your certificate will be a possibility as well.
 
 For additional reading on this process for trusting SSL certs locally, feel free to read up [here.](https://devblogs.microsoft.com/aspnet/configuring-https-in-asp-net-core-across-different-platforms/) The afore-mentioned [docker specific readme](../docker/README.md) also has details that are relevant.
+
+## Troubleshooting
+
+### Visual Studio - Community edition
+
+There's an issue with the Visual Studio Community edition. Use Enterprise to run locally.
```

Comment:
It would be helpful to explain what the issue is in case it is fixed in the future.

Reply:
I don't know what it is =D. It just throw an error about the app can start.

Reply:
![image](https://user-images.githubusercontent.com/24213737/152255208-acb9d1b5-de7b-4c80-a757-cc86f61d8c50.png)

Reply:
I just found out the issue 😄   updating pR in a moment